include/mptcpd/listener_manager.h

```diff
@@ -19,7 +19,7 @@
 extern "C" {
 #endif
 
-struct sockaddr;
+
```

Comment:
Extra newline?

Reply:
Yes.  I'll remove it in the next set of commits.

Reply:
Fixed in a99543f.